src/serac/physics/materials/thermal_functional_material.hpp

```diff
@@ -212,19 +263,6 @@ struct ConstantSource {
   }
```

Comment:
For line 208/259, little `t` is time? The quantities in `ParameterizedLinearIsotropicConductor` are not time dependent, so it's method don't have little `t`, right?

Reply:
I think `t` is time here. The material models probably don't get `t` because they don't usually depend explicitly on the absolute time (there are time-dependent materials, but the only ones I know of depend on a time increment, rather than the absolute time). Source terms, on the other hand, frequently are functions of the absolute time.